docs/index.html

```diff
@@ -745,7 +760,7 @@ <h2>Sign up to our Newsletter</h2>
           <form action="https://www.eclipse.org/donate/process.php" method="post" target="_blank">
             <div class="form-group">
               <input type="hidden" name="type" value="newsletter">
-              <input type="email" value="" name="email" class="textfield-underline form-control" id="mce-EMAIL" placeholder="Email">
+              <input type="email" value="" name="email" class="form-control" id="mce-EMAIL" placeholder="Email">
```

Comment:
No idea do we need this one? I don't see where it's used in our sites. Probably I'm missing Drupal sites?

Reply:
Just a note here, if I do not do anything else, just removed that class here, it becomes:

![image](https://user-images.githubusercontent.com/39588094/105080865-6eaa9c80-5a5f-11eb-9d85-37730de4805e.png)


But if this is not used anywhere, it's fine, we can remove it.